packages/data-point/README.md

```diff
@@ -1211,6 +1211,34 @@ constant(value:*):*
   ```
 </details>
```

Comment:
we should still keep the "Constants should not contain other reducers." sentence that you originally added

Reply:
Good idea

Reply:
I rather have it taken off or changed since the actual content of a constant could be later on used to build a reducer. The only fact is that constant will not evaluate their content, they can contain any type of value. So this comment is misleading.

Reply:
if we pass a reducer function as a constant, on a second iteration another reducer could use this value to execute itself


```js
'model:foo': {
   value: [
      { a: c((val) => val * 2) },
      (val) => val.a(2)
   ]
}
```

This only applies to Function Reducers, what I'm trying to articulate is just that constants can have anything, even a reducer if they want, just nothing will be evaluated, everything is taken as-is